packages/discovery-provider/integration_tests/challenges/test_audio_matching_challenges.py

```diff
@@ -17,6 +17,7 @@
 
 REDIS_URL = shared_config["redis"]["url"]
 BLOCK_NUMBER = 1
+BLOCK_DATETIME = datetime.now()
```

Comment:
doesn't this leave us with the same problem of if indexer is behind, the timestamps will not match across DNs?

Reply:
oh my bad this is a test file lol